packages/agoric-cli/src/install.js

```diff
@@ -169,8 +170,8 @@ export default async function installMain(progname, rawArgs, powers, opts) {
       )
         .then(results => {
           // After all have settled, throw any errors.
-          const failures = results.filter(
-            ({ status }) => status !== 'fulfilled',
+          const failures = /** @type {PromiseRejectedResult[]} */ (
+            results.filter(({ status }) => status !== 'fulfilled')
           );
```

Comment:
No cast necessary, but it only narrows if you don't destructure the status from the result:

```suggestion
          const failures = results.filter(result => result.status !== 'fulfilled');
```